### ISSUE TITLE
[QA] Screenreader

### ISSUE THREAD

**Issue by jessicasheng15** [vigetlabs/npm#147]
Note: I'm not very well-versed in screenreader testing so these may be the result of improper testing and/or typical items that crop up that don't necessarily need to be addressed.

**Audience**: https://live-national-public-media-v2.pantheonsite.io/audience/
It seems that the tabs being read are not the ones the user selects. For example: When a person using a screen reader clicks on 'Conscientious', the VoiceOver assistant reads 'Adventurous,' or typically the last tab that they clicked on. 
https://www.dropbox.com/s/gtjgyqt6rweyal4/Screenshot%202019-11-07%2017.46.57.png?dl=0

**Programs Index:** https://live-national-public-media-v2.pantheonsite.io/programs/
I don't think the screenreader is picking up content within modals. Currently, it will only read the NPR Podcast tag, the title (H1), and then float to the 'X' button. It does not read anything below the Program Title.
https://www.dropbox.com/s/6gv7cb7dwa5h576/Screenshot%202019-11-07%2017.48.05.png?dl=0

**Insights Index:** When a Learn More link below a preview card does not have text, VoiceOver assistant skips over it, and doesn't allow a user to click it. Screenshot below - it allowed me to select 'Learn More' but I couldn't read more about Test Article 8 or Test Article 7.
https://www.dropbox.com/s/94lpn7d2l62ki3w/Screenshot%202019-11-07%2017.38.45.png?dl=0

**Report Detail:** VoiceOver Assistant skips over embedded YouTube videos, won't allow me to play them.
https://www.dropbox.com/s/w1b52pyig5j4onb/Screenshot%202019-11-07%2017.41.30.png?dl=0

**Comment by jessicasheng15** [comment]
@justinvandenberg  In Programs index, the screen reader registers `button` or `collapsed, button` when tabbing through the Program cards _except_ for 1A and All Things Considered, for which it does announce the title. I would expect the screen reader to read the title for all cards (unexpanded).
https://www.dropbox.com/s/z0wsmg87kkoehjo/Screenshot%202019-11-19%2022.29.58.png?dl=0

Everything else listed in the ticket is resolved!

**Comment by jeremyfrank** [comment]
**Audience:** I was not able to replicate this issue.
**Programs Index:** It is working but you have to get the keyboard shortcut right to get it to start reading. I had to look it up: VO+A (VO=ctrl+opt). https://www.apple.com/voiceover/info/guide/_1131.html
**Insights Index:** I didn't see any read more links w/o text.
**Report Detail:** The YouTube player is known to not be accessible unfortunately.
